### ISSUE TITLE
Cannot get the IP on DSM 7.0 with haos_ova-6.2.ova

### ISSUE THREAD

**Issue by griffinqiu** [home-assistant/operating-system#1499]
![image](https://user-images.githubusercontent.com/179343/128542587-3cd287f5-71ef-4cf7-b33b-d3ac043c9402.png)

**Comment by agners** [comment]
Not familiar with DSM, what is that? What kind of Hypervisor are we talking about? What settings (specifically for network) are you using in your Hypervisor configuration?

**Comment by Giga-Pudding** [comment]
I have the same Problem, using ESXi Hypervisor. I am not able to configure IP Settings.
Also, the command `--ipv4-method static` causes an error. Something is broken here.

![ha_error](https://user-images.githubusercontent.com/56741807/129680702-56ecdf51-90ab-42a9-8d2f-37359fe180b2.png)

**Comment by Giga-Pudding** [comment]
> Not familiar with DSM, what is that? What kind of Hypervisor are we talking about? What settings (specifically for network) are you using in your Hypervisor configuration?

When @griffinqiu wrote "DSM 7.0" i think he meant "DiskStation Manager" which is the name of the Operating System of Synology NAS devices. DSM 7.0 is the latest release. It also contains a hypervisor (KVM, as far as i know).

**Comment by 47780** [comment]
I am running into the exact same issue, I am on 6.2 and when I try to set the IP to static I get the same error. My hypervisor is proxmox.

**Comment by 47780** [comment]
> I am running into the exact same issue, I am on 6.2 and when I try to set the IP to static I get the same error. My hypervisor is proxmox.

I did get the network working by following the nmcli instructions [here](https://github.com/home-assistant/operating-system/blob/rel-6/Documentation/network.md) to set a static IP. The only difference is that I made the changes to the supervisor config instead of the "Home Assistant OS default" config. Changes to "Home Assistant OS default" didn't work.
